src/v2/coins/eth.ts

```diff
@@ -321,7 +329,7 @@ class Eth extends BaseCoin {
       throw new Error('missing prv parameter to sign transaction');
     }
 
-    params.recipients = params.recipients || txPrebuild.recipients;
+    params.recipients = txPrebuild.recipients || params.recipients;
```

Comment:
Needed to switch the preference order here because the recipients from the prebuild are more accurate than the originally requested recipients in this case

Reply:
This is done, right? Double checking.